image_shot:
  # rgbTopicName
  rgbTopicName: /camera/color/image_raw
  # depthTopicName
  depthTopicName: /camera/aligned_depth_to_color/image_raw #/camera/depth/image_rect_raw
  # rgb_prefix
  rgb_prefix: rgb_
  # depth_prefix
  depth_prefix: dep_
  # data_set
  data_set: test
  # delta_stamp #@1ms
  delta_stamp: 0.001 
  # queue_depth
  queue_depth: 100
  # control_frequency
  control_frequency: 50.0


